ext/Geant4/g4jl_application.jl

```diff
@@ -0,0 +1,144 @@
+struct SSDPhysics <: G4VUserPhysicsList
+    function SSDPhysics(verbose)
+        pl = G4VModularPhysicsList()
+        RegisterPhysics(pl, move!(G4DecayPhysics(verbose)))             # Default physics
+        RegisterPhysics(pl, move!(G4EmStandardPhysics(verbose)))        # EM physics
+        RegisterPhysics(pl, move!(G4RadioactiveDecayPhysics(verbose)))  # Radioactive decay
+        return pl
+    end
+end
+
+struct SDData{T} <: G4JLSDData
+  detectorHits::DetectorHits
+  SDData{T}() where {T} = new(DetectorHits((evtno = Int32[], detno = Int32[], thit = typeof(zero(T)*u"s")[], edep = typeof(zero(T)*u"keV")[], pos = SVector{3,typeof(zero(T)*u"mm")}[])))
+end
+
+function _initialize(::G4HCofThisEvent, data::SDData)::Nothing
+  empty!(data.detectorHits)
+  return
+end
+
+function _endOfEvent(::G4HCofThisEvent, data::SDData)::Nothing
+  return
+end
+
+function _processHits(step::G4Step, ::G4TouchableHistory, data::SDData{T})::Bool where {T}
+    edep = step |> GetTotalEnergyDeposit
+    iszero(edep) && return false
+    pos = step |> GetPostStepPoint |> GetPosition
+    push!(data.detectorHits, 
+        (
+            # evtno
+            evtno = Int32(0),
+            # detno
+            detno = Int32(1), #step |> GetPostStepPoint |> GetPhysicalVolume, # get ID?
+            # thit
+            thit = T(0)*u"s", #step |> GetPostStepPoint |> GetGlobalTime,
```

Comment:
The fields `evtno`, `detno`  and `thit` are currently set arbitrarily. Further changes are needed to store the correct values.